libs/components/lookup/src/lib/modules/lookup/lookup.component.spec.ts

```diff
@@ -3744,11 +3744,26 @@ describe('Lookup component', function () {
 
       it('should be accessible', async () => {
         fixture.componentInstance.ariaLabelledBy = 'my-lookup-label';
+        fixture.componentInstance.setSingleSelect();
```

Comment:
Since these changes involve the lookup library, I would put them in a separate pull request so that they're mentioned in the changelog.

Reply:
Removed the autocomplete changes.